src/common/options/global.yaml.in

```diff
@@ -5030,6 +5030,13 @@ options:
     This setting is used only when OSD is doing ``--mkfs``.
     Next runs of OSD retrieve sharding from disk.
   default: m(3) p(3,0-12) O(3,0-13)=block_cache={type=binned_lru} L=min_write_buffer_number_to_merge=32 P=min_write_buffer_number_to_merge=32
+- name: bluestore_async_db_compaction
```

Comment:
I would prefer this option to control compaction method on initialization.
The way of compacting via admin socket command would be passed in the command.